OCPBUGS 43007: Removed the acme.cert-manager.io/http01-ingress-class annotation (#87341)
openshift/openshift-docs: modules/cert-manager-acme-http01.adoc

  namespace: my-ingress-namespace                                <2>
  annotations:
    cert-manager.io/cluster-issuer: letsencrypt-staging          <3>
spec:
  ingressClassName: openshift-default                            <4>
  tls:
  - hosts:
    - <hostname>                                                 <5>
    secretName: sample-tls                                       <6>
  rules:
  - host: <hostname>                                             <7>
    http:
      paths:
      - path: /
        pathType: Prefix
        backend:
          service:
            name: sample-workload                                <8>
            port:
              number: 80
----
<1> Specify the name of the Ingress.
<2> Specify the namespace that you created for the Ingress.
<3> Specify the cluster issuer that you created.
<4> Specify the Ingress class.

> <5> Replace `<hostname>` with the Subject Alternative Name to be associated with the certificate. This name is used to add DNS names to the certificate.

Reviewer: Does "Subject Alternative Name" need to be in Caps?

> <5> Replace `<hostname>` with the Subject Alternative Name to be associated with the certificate. This name is used to add DNS names to the certificate.

Reviewer: I see "Subject Alternative Name (SAN)" in other docs. Consider using the acronym.

> <6> Specify the secret to store the created certificate in.

Reviewer: ```suggestion
<6> Specify the secret that stores the certificate.
```
